airbyte-integrations/connectors/source-mssql/src/main/java/io/airbyte/integrations/source/mssql/MssqlSource.java

```diff
@@ -456,42 +449,6 @@ protected void assertSqlServerAgentRunning(final JdbcDatabase database) throws S
     return super.getIncrementalIterators(database, catalog, tableNameToTable, stateManager, emittedAt);
   }
 
-  public AutoCloseableIterator<AirbyteMessage> getDebeziumSnapshotIterators(
```

Comment:
is it just this function was deprecated or never used?

Reply:
yes it's never used - we used to rely on DBZ for snapshotting, but now we perform it ourselves with the chunking queries